prisma/migrations/20210710225536_added_reviews/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "Review" (
+    "id" SERIAL NOT NULL,
+    "movie_id" TEXT NOT NULL,
+    "author_id" INTEGER NOT NULL,
+    "content" TEXT,
+    "rating" DOUBLE PRECISION NOT NULL,
+    "created_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMP(3) NOT NULL,
+
+    PRIMARY KEY ("id")
+);
+
+-- AddForeignKey
+ALTER TABLE "Review" ADD FOREIGN KEY ("author_id") REFERENCES "User"("id") ON DELETE CASCADE ON UPDATE CASCADE;
```

Comment:
So if a user is deleted, all Reviews by the `author_id` == `id` will also be deleted right?

Reply:
Yip that's correct. I would imagine we don't want any orphan reviews? i.e. every review should be owned by a valid user

Reply:
Good data purging practice. Respect.